amy/dashboard/views.py

```diff
@@ -442,6 +443,15 @@ def form_valid(self, form):
                 f"{', '.join(gen)}",
             )
 
+        instructor_signs_up_for_workshop_signal.send(
```

Comment:
Good catch, I didn't think of this at all. I'll make an issue to fix this somehow.

Reply:
Issue created: https://github.com/carpentries/amy/issues/2498

This is currently the only case of user-triggered (instead of admin-triggered) action, so I'm wondering how to approach this problem.